advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Installing EDB Wait States
+navTitle: Installing
+---
+
+## On Linux 
+
+To install EWSBW on a Linux system:
+
+1.  Install EDB wait states with the `edb-asxx-server-edb_wait_states` RPM package, where `xx` is the EDB Postgres Advanced Server version number.
+
+1. To launch the worker, register it in the `postgresql.conf` file using the `shared_preload_libraries` parameter. For example:
+
+  ```ini
+  shared_preload_libraries = '$libdir/edb_wait_states'
+  ```
+
+1.  Restart the database server. After a successful restart, the background worker begins collecting data.
+
+1. To review the data, create the following extension:
+
+  ```sql
+  CREATE EXTENSION edb_wait_states;
+  ```
+
+1. To terminate the EDB wait states worker, remove `$libdir/edb_wait_states` from the `shared_preload_libraries` parameter, and restart the database server.
+
+## On Windows
+
+To install EWSBW on a Windows system:
+
+1.  To install the EDB wait states module with the `EDB Modules` installer, invoke the StackBuilder Plus utility. Follow the onscreen instructions to finish installing the `EDB Modules`.
+
+1. To register the worker, modify the `postgresql.conf` file to include the wait states library in the `shared_preload_libraries` configuration parameter. The parameter value must include:
+
+  ```ini
+  shared_preload_libraries = '$libdir/edb_wait_states.dll'
+  ```
+
+  The EDB wait states installation places the `edb_wait_states.dll` library file in the following path:
+
+  ```text
+  C:\Program Files\edb\as14\lib\
+  ```
+
+1. Restart the database server to make the changes take effect. After a successful restart, the background worker starts and begins collecting the data.
+
+1. To view the data, create the following extension:
+
+   ```sql
+   CREATE EXTENSION edb_wait_states;
+   ```
+
+The installer places the `edb_wait_states.control` file in the following path:
+
+```text
+C:\Program Files\edb\as14\share\extension
+```
+
+## Terminating
```

Comment:
Can this just be Uninstalling instead of Terminating? Or is it distinct.

Reply:
This does appear to be uninstalling the extension, especially as it requires a server restart. I would rename it.

Reply:
Are we talking about disabling the extension? If yes, we recently commit DB-1962, which allows a user to disable the edb_wait_state without a restart.  Here is the particular commit:

commit 94cfc0ef78577817638ec210492f2694529ee2c1
Author: Rushabh Lathia <rushabh.lathia@enterprisedb.com>
Date:   Tue Nov 8 17:31:25 2022 +0530

    Add GUC edb_wait_states.enable_collection to Enable/Disable collecting
    wait states data without a database server restart.

Reply:
This feature suggests the the docs should be reorganised around...

* Installing
* Using
  * Activating
  * Exploring the data through the interface
  * Deactivating
* Uninstalling

With collection disabled, @lathiarb the interface and already collected data will be accessible yes?

Reply:
@lathiarb Is `edb_wait_states.enable_collection` automatically set to `ON` when EDB Wait States is installed?

Reply:
By default, it is enabled. also, we have NOT released build which contain this GUC parameter so please don't write it in the DOC

Reply:
edb=# show edb_wait_states.enable_collection ;
 edb_wait_states.enable_collection 
-----------------------------------
 on
(1 row)

edb=#